Warnings during compilation in develop branch

I'm getting some warning during compilation using latest develop branch:
```
../components/lvgl_esp32_drivers/lvgl_tft/ili9481.c: In function 'ili9481_flush':
../components/lvgl_esp32_drivers/lvgl_tft/ili9481.c:117:81: warning: suggest braces around empty body in an 'if' statement [-Wempty-body]
         if (mybuf == NULL)  LV_LOG_WARN("Could not allocate enough DMA memory!");
                                                                                 ^
../components/lvgl_esp32_drivers/lvgl_tft/ili9481.c: In function 'ili9481_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/ili9481.c:192:17: warning: unused variable 'orientation_str' [-Wunused-variable]
     const char *orientation_str[] = {
                 ^~~~~~~~~~~~~~~
[1105/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/ili9486.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/ili9486.c: In function 'ili9486_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/ili9486.c:168:17: warning: unused variable 'orientation_str' [-Wunused-variable]
     const char *orientation_str[] = {
                 ^~~~~~~~~~~~~~~
[1107/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/st7735s.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/st7735s.c: In function 'st7735s_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/st7735s.c:208:17: warning: unused variable 'orientation_str' [-Wunused-variable]
     const char *orientation_str[] = {
                 ^~~~~~~~~~~~~~~
[1110/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/st7796s.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/st7796s.c: In function 'st7796s_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/st7796s.c:192:14: warning: unused variable 'orientation_str' [-Wunused-variable]
  const char *orientation_str[] = {
              ^~~~~~~~~~~~~~~
[1112/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/ili9488.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/ili9488.c: In function 'ili9488_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/ili9488.c:192:17: warning: unused variable 'orientation_str' [-Wunused-variable]
     const char *orientation_str[] = {
                 ^~~~~~~~~~~~~~~
[1117/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/GC9A01.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/GC9A01.c: In function 'GC9A01_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/GC9A01.c:224:17: warning: unused variable 'orientation_str' [-Wunused-variable]
     const char *orientation_str[] = {
                 ^~~~~~~~~~~~~~~
[1119/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/ili9163c.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/ili9163c.c: In function 'ili9163c_set_orientation':
../components/lvgl_esp32_drivers/lvgl_tft/ili9163c.c:226:14: warning: unused variable 'orientation_str' [-Wunused-variable]
  const char *orientation_str[] = {
              ^~~~~~~~~~~~~~~
[1120/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/jd79653a.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/jd79653a.c: In function 'jd79653a_lv_fb_flush':
../components/lvgl_esp32_drivers/lvgl_tft/jd79653a.c:412:12: warning: unused variable 'len' [-Wunused-variable]
     size_t len = ((area->x2 - area->x1 + 1) * (area->y2 - area->y1 + 1)) / 8;
            ^~~
At top level:
../components/lvgl_esp32_drivers/lvgl_tft/jd79653a.c:179:13: warning: 'jd79653a_spi_send_fb' defined but not used [-Wunused-function]
 static void jd79653a_spi_send_fb(uint8_t *data, size_t len)
             ^~~~~~~~~~~~~~~~~~~~
[1121/1130] Building C object esp-idf/lvgl_esp32_drivers/CMakeFiles/__idf_lvgl_esp32_drivers.dir/lvgl_tft/uc8151d.c.obj
../components/lvgl_esp32_drivers/lvgl_tft/uc8151d.c: In function 'uc8151d_lv_fb_flush':
../components/lvgl_esp32_drivers/lvgl_tft/uc8151d.c:199:12: warning: unused variable 'len' [-Wunused-variable]
     size_t len = ((area->x2 - area->x1 + 1) * (area->y2 - area->y1 + 1)) / 8;
            ^~~
At top level:
../components/lvgl_esp32_drivers/lvgl_tft/uc8151d.c:104:13: warning: 'uc8151d_spi_send_seq' defined but not used [-Wunused-function]
 static void uc8151d_spi_send_seq(const uc8151d_seq_t *seq, size_t len)
             ^~~~~~~~~~~~~~~~~~~~
../components/lvgl_esp32_drivers/lvgl_tft/uc8151d.c:97:13: warning: 'uc8151d_spi_send_fb' defined but not used [-Wunused-function]
 static void uc8151d_spi_send_fb(uint8_t *data, size_t len)
             ^~~~~~~~~~~~~~~~~~~
```

--- a/lvgl_tft/GC9A01.c
+++ b/lvgl_tft/GC9A01.c
@@ -221,12 +221,6 @@ static void GC9A01_set_orientation(uint8_t orientation)
 {
     assert(orientation < 4);
 
-    const char *orientation_str[] = {
-        "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
-    };
-
-    LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
-
 #if defined CONFIG_LV_PREDEFINED_DISPLAY_M5STACK
     const uint8_t data[] = {0x68, 0x68, 0x08, 0x08};
 #elif defined (CONFIG_LV_PREDEFINED_DISPLAY_WROVER4)
@@ -235,7 +229,14 @@ static void GC9A01_set_orientation(uint8_t orientation)
     const uint8_t data[] = {0x08, 0xC8, 0x68, 0xA8};
 #endif
 
+#if (LV_USE_LOG == 1);
+    const char *orientation_str[] = {
+        "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
+    };
+
+    LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
     LV_LOG_INFO("0x36 command value: 0x%02X", data[orientation]);
+#endif
 
     GC9A01_send_cmd(0x36);
     GC9A01_send_data((void *) &data[orientation], 1);

--- a/lvgl_tft/ili9163c.c
+++ b/lvgl_tft/ili9163c.c
@@ -223,11 +223,12 @@ static void ili9163c_set_orientation(uint8_t orientation)
 {
 	assert(orientation < 4);
 
+#if (LV_USE_LOG == 1)
 	const char *orientation_str[] = {
 		"PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"};
 
 	LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
-
+#endif
 	uint8_t data[] = {0x48, 0x88, 0xA8, 0x68};
 
 	ili9163c_send_cmd(ILI9163C_MADCTL);

--- a/lvgl_tft/ili9481.c
+++ b/lvgl_tft/ili9481.c
@@ -114,7 +114,9 @@ void ili9481_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * col
     uint8_t *mybuf;
     do {
         mybuf = (uint8_t *) heap_caps_malloc(3 * size * sizeof(uint8_t), MALLOC_CAP_DMA);
-        if (mybuf == NULL)  LV_LOG_WARN("Could not allocate enough DMA memory!");
+        if (mybuf == NULL)  {
+            LV_LOG_WARN("Could not allocate enough DMA memory!");
+        }
     } while (mybuf == NULL);
 
     uint32_t LD = 0;
@@ -189,11 +191,13 @@ static void ili9481_send_color(void * data, uint16_t length)
 
 static void ili9481_set_orientation(uint8_t orientation)
 {
+#if (LV_USE_LOG == 1)
     const char *orientation_str[] = {
         "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
     };
 
     LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
+#endif
 
     uint8_t data[] = {0x48, 0x4B, 0x28, 0x2B};
     ili9481_send_cmd(ILI9481_CMD_MEMORY_ACCESS_CONTROL);

--- a/lvgl_tft/ili9486.c
+++ b/lvgl_tft/ili9486.c
@@ -165,15 +165,16 @@ static void ili9486_set_orientation(uint8_t orientation)
 {
     assert(orientation < 4);
 
+    const uint8_t data[] = {0x48, 0x88, 0x28, 0xE8};
+
+#if (LV_USE_LOG == 1)
     const char *orientation_str[] = {
         "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
     };
 
     LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
-
-    const uint8_t data[] = {0x48, 0x88, 0x28, 0xE8};
-
     LV_LOG_INFO("0x36 command value: 0x%02X", data[orientation]);
+#endif
 
     ili9486_send_cmd(0x36);
     ili9486_send_data((void *) &data[orientation], 1);

--- a/lvgl_tft/ili9488.c
+++ b/lvgl_tft/ili9488.c
@@ -188,16 +188,16 @@ static void ili9488_send_color(lv_disp_drv_t * drv, void * data, uint16_t length
 static void ili9488_set_orientation(lv_disp_drv_t * drv, uint8_t orientation)
 {
     assert(orientation < 4);
+    const uint8_t data[] = {0x48, 0x88, 0x28, 0xE8};
 
+#if (LV_USE_LOG == 1)
     const char *orientation_str[] = {
         "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
     };
 
     LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
-
-    const uint8_t data[] = {0x48, 0x88, 0x28, 0xE8};
-
     LV_LOG_INFO("0x36 command value: 0x%02X", data[orientation]);
+#endif
 
     ili9488_send_cmd(drv, 0x36);
     ili9488_send_data(drv, (void *) &data[orientation], 1);

--- a/lvgl_tft/jd79653a.c
+++ b/lvgl_tft/jd79653a.c
@@ -176,13 +176,6 @@ static void jd79653a_spi_send_data(uint8_t *data, size_t len)
     disp_spi_send_data(data, len);
 }
 
-static void jd79653a_spi_send_fb(uint8_t *data, size_t len)
-{
-    disp_wait_for_pending_transactions();
-    gpio_set_level(PIN_DC, 1);   // DC = 1 for data
-    disp_spi_send_colors(data, len);
-}
-
 static void jd79653a_spi_send_seq(const jd79653a_seq_t *seq, size_t len)
 {
     LV_LOG_INFO("Writing cmd/data sequence, count %u", len);
@@ -409,8 +402,6 @@ void jd79653a_lv_rounder_cb(struct _disp_drv_t *disp_drv, lv_area_t *area)
 
 void jd79653a_lv_fb_flush(lv_disp_drv_t *drv, const lv_area_t *area, lv_color_t *color_map)
 {
-    size_t len = ((area->x2 - area->x1 + 1) * (area->y2 - area->y1 + 1)) / 8;
-
     LV_LOG_INFO("x1: 0x%x, x2: 0x%x, y1: 0x%x, y2: 0x%x", area->x1, area->x2, area->y1, area->y2);
     LV_LOG_INFO("Writing LVGL fb with len: %u, partial counter: %u", len, partial_counter);
 

--- a/lvgl_tft/st7735s.c
+++ b/lvgl_tft/st7735s.c
@@ -205,11 +205,7 @@ static void st7735s_send_color(void * data, uint16_t length)
 
 static void st7735s_set_orientation(uint8_t orientation)
 {
-    const char *orientation_str[] = {
-        "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
-    };
-
-    LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
+    assert(orientation < 4);
 
     /*
         Portrait:  0xC8 = ST77XX_MADCTL_MX | ST77XX_MADCTL_MY | ST77XX_MADCTL_BGR
@@ -218,7 +214,14 @@ static void st7735s_set_orientation(uint8_t orientation)
     */
     uint8_t data[] = {0xC8, 0xC8, 0xA8, 0xA8};
 
+#if (LV_USE_LOG == 1)
+    const char *orientation_str[] = {
+        "PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"
+    };
+
+    LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
     LV_LOG_INFO("0x36 command value: 0x%02X", data[orientation]);
+#endif
 
     st7735s_send_cmd(ST7735_MADCTL);
     st7735s_send_data((void *) &data[orientation], 1);

--- a/lvgl_tft/st7796s.c
+++ b/lvgl_tft/st7796s.c
@@ -189,11 +189,6 @@ static void st7796s_set_orientation(uint8_t orientation)
 {
 	assert(orientation < 4);
 
-	const char *orientation_str[] = {
-		"PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"};
-
-	LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
-
 #if defined CONFIG_LV_PREDEFINED_DISPLAY_M5STACK
 	const uint8_t data[] = {0x68, 0x68, 0x08, 0x08};
 #elif defined(CONFIG_LV_PREDEFINED_DISPLAY_WROVER4)
@@ -204,7 +199,13 @@ static void st7796s_set_orientation(uint8_t orientation)
 	const uint8_t data[] = {0x48, 0x88, 0x28, 0xE8};
 #endif
 
+#if (LV_USE_LOG == 1)
+	const char *orientation_str[] = {
+		"PORTRAIT", "PORTRAIT_INVERTED", "LANDSCAPE", "LANDSCAPE_INVERTED"};
+
+	LV_LOG_INFO("Display orientation: %s", orientation_str[orientation]);
 	LV_LOG_INFO("0x36 command value: 0x%02X", data[orientation]);
+#endif
 
 	st7796s_send_cmd(0x36);
 	st7796s_send_data((void *)&data[orientation], 1);

--- a/lvgl_tft/uc8151d.c
+++ b/lvgl_tft/uc8151d.c
@@ -94,26 +94,6 @@ static void uc8151d_spi_send_data_byte(uint8_t data)
     disp_spi_send_data(&data, 1);
 }
 
-static void uc8151d_spi_send_fb(uint8_t *data, size_t len)
-{
-    disp_wait_for_pending_transactions();
-    gpio_set_level(PIN_DC, 1);   // DC = 1 for data
-    disp_spi_send_colors(data, len);
-}
-
-static void uc8151d_spi_send_seq(const uc8151d_seq_t *seq, size_t len)
-{
-    LV_LOG_INFO("Writing cmd/data sequence, count %u", len);
-
-    if (!seq || len < 1) return;
-    for (size_t cmd_idx = 0; cmd_idx < len; cmd_idx++) {
-        uc8151d_spi_send_cmd(seq[cmd_idx].cmd);
-        if (seq[cmd_idx].len > 0) {
-            uc8151d_spi_send_data((uint8_t *) seq[cmd_idx].data, seq[cmd_idx].len);
-        }
-    }
-}
-
 static esp_err_t uc8151d_wait_busy(uint32_t timeout_ms)
 {
     uint32_t wait_ticks = (timeout_ms == 0 ? portMAX_DELAY : pdMS_TO_TICKS(timeout_ms));
@@ -196,8 +176,6 @@ static void uc8151d_full_update(uint8_t *buf)
 
 void uc8151d_lv_fb_flush(lv_disp_drv_t *drv, const lv_area_t *area, lv_color_t *color_map)
 {
-    size_t len = ((area->x2 - area->x1 + 1) * (area->y2 - area->y1 + 1)) / 8;
-
     LV_LOG_INFO("x1: 0x%x, x2: 0x%x, y1: 0x%x, y2: 0x%x", area->x1, area->x2, area->y1, area->y2);
     LV_LOG_INFO("Writing LVGL fb with len: %u", len);
 

